docs/sync_code.md

```diff
@@ -0,0 +1,34 @@
+# Basic workflow for Operator
+
+1. Create a branch:
+Start by creating a branch from "main" branch.
+
+2. Ensure local tests pass:
+Run unit and e2e tests, and ensure they pass before submitting a pull request (PR) targeting "main" branch.
```

Comment:
will we switch to jira-XXX-name branch naming at the same time?

Reply:
the branch is created in the users' fork git repo, can be named any way they would like to. i do not think we need suggest that part.